tests/search/components/TestSearchResults.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import SearchResults from 'search/components/SearchResults';
+
+const render = ReactTestUtils.renderIntoDocument;
+const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
+const findByClass = ReactTestUtils.findRenderedDOMComponentWithClass;
+const isDOMComponent = ReactTestUtils.isDOMComponent;
+
+
+describe('<SearchResults />', () => {
+  function renderResults(props) {
+    return render(<SearchResults {...props} />);
+  }
+
+  it('renders empty search results container', () => {
+    const root = renderResults();
+    const searchResults = findByClass(root, 'search-results');
+    assert.ok(isDOMComponent(searchResults));
+    assert.equal(searchResults.childNodes.length, 0);
+  });
+
+  it('renders error when query is an empty string', () => {
+    const root = renderResults({query: ''});
+    const searchResultsMsg = findByTag(root, 'p');
+    assert.include(searchResultsMsg.firstChild.nodeValue, 'supply a valid search');
+  });
+
+  it('renders error when no results and valid query', () => {
+    const root = renderResults({query: 'test'});
+    const searchResultsMsg = findByTag(root, 'p');
```

Comment:
I think I like the idea of using `refs` for stuff like this. Instead of looking up the `<p>` we could look up `root.refs.message` and then we don't care what the markup is.